### DISCUSSION TITLE
v11.0.0 | 1.19.3

### DISCUSSION THREAD

**Discussion by TheMrJezza** [TheMrJezza/HorseTpWithMe#17] [General]
Upgrade to 1.19.3

<hr /><em>This discussion was created from the release <a href='https://github.com/TheMrJezza/HorseTpWithMe/releases/tag/Release'>v11.0.0 | 1.19.3</a>.</em>

**Comment by CO0KlE** [comment]
Hey man, any chance to get update for 1.20.6?
